src/triagebot/canonicalize-issue-links.md

```diff
@@ -1,6 +1,7 @@
 # Canonicalize Issue Links
 
-GitHub permits having automatic action like `Fixes #123` which closes the issue number `123`, this handler updates the pull-request description with the canonicalized version, `Fixes org/repo#123`.
+GitHub permits having automatic action like `Fixes #123`, which closes the issue number `123`.
+This handler updates the pull-request description with the canonicalized version, `Fixes org/repo#123`.
```

Comment:
Unless I'm missing something, those two lines are still displayed on the same line. Was that your intention? If not, you need an empty line in between.

Reply:
the change splits the sentence into 2, and adds a pause after the first "Fixes"